packages/mobile/src/setupE2eEnv.e2e.ts

```diff
@@ -0,0 +1,26 @@
+import Config from 'react-native-config'
```

Comment:
What causes this `.e2e.ts` file to be imported instead of the other ts file? Is that a Detox thing?

Reply:
It's activated in `metro.config.js` with the env var `CELO_TEST_CONFIG=e2e`.